# Translations template for ironic-inspector.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the ironic-inspector
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2015.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: ironic-inspector 2.0.0.dev179\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-07-08 21:42+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: ironic_inspector/introspect.py:38
msgid "IPMI credentials setup is disabled in configuration"
msgstr ""

#: ironic_inspector/introspect.py:42
msgid "Node should be in maintenance mode to set IPMI credentials on it"
msgstr ""

#: ironic_inspector/introspect.py:49
#, python-format
msgid ""
"Setting IPMI credentials requested for node %s, but neither new user name"
" nor driver_info[ipmi_username] are provided"
msgstr ""

#: ironic_inspector/introspect.py:56
#, python-format
msgid ""
"Forbidden characters encountered in new IPMI password for node %(node)s: "
"\"%(chars)s\"; use only letters and numbers"
msgstr ""

#: ironic_inspector/introspect.py:61
#, python-format
msgid "IPMI password length should be > 0 and <= %d"
msgstr ""

#: ironic_inspector/introspect.py:79
#, python-format
msgid "Cannot find node %s"
msgstr ""

#: ironic_inspector/introspect.py:81
#, python-format
msgid "Cannot get node %(node)s: %(exc)s"
msgstr ""

#: ironic_inspector/introspect.py:92
#, python-format
msgid "Failed validation of power interface for node %(node)s, reason: %(reason)s"
msgstr ""

#: ironic_inspector/introspect.py:107
msgid "Unexpected exception in background introspection thread"
msgstr ""

#: ironic_inspector/introspect.py:137
#, python-format
msgid ""
"Failed to power on node %(node)s, check it's power management "
"configuration:\n"
"%(exc)s"
msgstr ""

#: ironic_inspector/main.py:79
msgid "Invalid UUID value"
msgstr ""

#: ironic_inspector/node_cache.py:115
#, python-format
msgid "Some or all of %(name)s's %(value)s are already on introspection"
msgstr ""

#: ironic_inspector/node_cache.py:269
#, python-format
msgid "Could not find node %s in cache"
msgstr ""

#: ironic_inspector/node_cache.py:304
#, python-format
msgid "Could not find a node for attributes %s"
msgstr ""

#: ironic_inspector/node_cache.py:307
#, python-format
msgid "Multiple matching nodes found for attributes %(attr)s: %(found)s"
msgstr ""

#: ironic_inspector/node_cache.py:317
#, python-format
msgid ""
"Could not find node %s in introspection cache, probably it's not on "
"introspection now"
msgstr ""

#: ironic_inspector/node_cache.py:322
#, python-format
msgid "Introspection for node %(node)s already finished on %(finish)s"
msgstr ""

#: ironic_inspector/process.py:42 ironic_inspector/process.py:56
#, python-format
msgid "Look up error: %s"
msgstr ""

#: ironic_inspector/process.py:52
msgid "Node not found hook returned nothing"
msgstr ""

#: ironic_inspector/process.py:54
#, python-format
msgid "Node not found hook failed: %s"
msgstr ""

#: ironic_inspector/process.py:81
#, python-format
msgid "Unexpected exception during preprocessing in hook %s"
msgstr ""

#: ironic_inspector/process.py:87
#, python-format
msgid ""
"The following failures happened during running pre-processing hooks for "
"node %(uuid)s:\n"
"%(failures)s"
msgstr ""

#: ironic_inspector/process.py:92
msgid "Data pre-processing failed"
msgstr ""

#: ironic_inspector/process.py:95
#, python-format
msgid ""
"The following failures happened during running pre-processing hooks for "
"unknown node:\n"
"%(failures)s"
msgstr ""

#: ironic_inspector/process.py:105
#, python-format
msgid "Node UUID %s was found in cache, but is not found in Ironic"
msgstr ""

#: ironic_inspector/process.py:116
msgid "Unexpected exception during processing"
msgstr ""

#: ironic_inspector/process.py:201
#, python-format
msgid ""
"Failed to validate updated IPMI credentials for node %s, node might "
"require maintenance"
msgstr ""

#: ironic_inspector/process.py:213
#, python-format
msgid ""
"Failed to power off node %(node)s, check it's power management "
"configuration: %(exc)s"
msgstr ""

#: ironic_inspector/utils.py:119
msgid "Authentication required"
msgstr ""

#: ironic_inspector/utils.py:123
msgid "Access denied"
msgstr ""

#: ironic_inspector/utils.py:174
#, python-format
msgid ""
"Refusing to introspect node %(node)s with provision state \"%(state)s\" "
"and maintenance mode off"
msgstr ""

#: ironic_inspector/common/swift.py:110
#, python-format
msgid "Swift failed to create container %(container)s. Error was: %(error)s"
msgstr ""

#: ironic_inspector/common/swift.py:125
#, python-format
msgid ""
"Swift failed to create object %(object)s in container %(container)s. "
"Error was: %(error)s"
msgstr ""

#: ironic_inspector/plugins/standard.py:45
#, python-format
msgid "The following required parameters are missing: %s"
msgstr ""

#: ironic_inspector/plugins/standard.py:85
msgid "No interfaces supplied by the ramdisk"
msgstr ""

#: ironic_inspector/plugins/standard.py:112
#, python-format
msgid ""
"No valid interfaces found for node with BMC %(ipmi_address)s, got "
"%(interfaces)s"
msgstr ""

#: ironic_inspector/plugins/standard.py:171
#, python-format
msgid "Ramdisk reported error: %s"
msgstr ""

